### PR TITLE
Spanish: Serving Ads on AMP

### PR DESCRIPTION
**Pull request by SidVal**
New files :es: 
* `ads_getting_started@es.md`
* `ads_on_amp@es.md`
* `ads_tips@es.md`

### CODE REVIEW

File: content/docs/guides/ads_on_amp/ads_getting_started@es.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      height="200"
+      type="doubleclick"
+      data-slot="/4119129/doesnt-exist">
+    <amp-img fallback src="fallback-image.jpg"></amp-img>
+  </amp-ad>
+[/sourcecode]
+
+AMP admite un atributo opcional de fallback. Dependiendo de la red de anuncios, puede elegir mostrar este elemento alternativo si no hay ningún anuncio disponible para publicar.
+
+{% call callout('Nota', type='note') %}
```
Comment (bpaduch) [inline content/docs/guides/ads_on_amp/ads_getting_started@es.md]: This note isn't in the English version, is this intentional? Typically they match.

File: content/docs/guides/ads_on_amp/ads_getting_started@es.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      height="200"
+      type="doubleclick"
+      data-slot="/4119129/doesnt-exist">
+    <amp-img placeholder src="placeholder-image.jpg"></amp-img>
+  </amp-ad>
+[/sourcecode]
+
+AMP admite un atributo de marcador de posición opcional. Dependiendo de la red de anuncios, puede elegir mostrar un marcador de posición hasta que el anuncio esté disponible para su visualización. Esto provee una mejor experiencia del usuario previniendo que se muestre un espacio en blanco.
+
+{% call callout('Nota', type='note') %}
```
Comment (bpaduch) [inline content/docs/guides/ads_on_amp/ads_getting_started@es.md]: This note isn't in the English version, is this intentional? Typically they match.

File: content/docs/guides/ads_on_amp/ads_tips@es.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+---
+$title : Tips para Anuncios sobre páginas AMP
+$order : 1
+---
+
+### Colocación & Controles
```
Comment (bpaduch) [inline content/docs/guides/ads_on_amp/ads_tips@es.md]: I noticed the formatting on these headings differ from the English version.  Intentional?

File: content/docs/guides/ads_on_amp/ads_getting_started@es.md (hunk 2 above)
Comment (SidVal) [inline content/docs/guides/ads_on_amp/ads_getting_started@es.md]: Yes, I added notes for a better understanding of the text. I know they are not in English, I would propose a issue to improve the file in English but then I forgot it.

Do you want to leave the note or I remove it?

File: content/docs/guides/ads_on_amp/ads_getting_started@es.md (hunk 1 above)
Comment (SidVal) [inline content/docs/guides/ads_on_amp/ads_getting_started@es.md]: Yes, I added notes for a better understanding of the text. I know they are not in English, I would propose a issue to improve the file in English but then I forgot it.

Do you want to leave the note or I remove it?

File: content/docs/guides/ads_on_amp/ads_tips@es.md (hunk 3 above)
Comment (SidVal) [inline content/docs/guides/ads_on_amp/ads_tips@es.md]: Intentional.

The `###` in markdown is for headlines `<h3>` in this case, and these headlines with `<br />` ? why a `<br />` tag into a headline? :-1:

File: content/docs/guides/ads_on_amp@es.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    <li><a href="/es/support/faqs/supported-platforms#ads">Plataformas aceptadas</a></li>
+    <li><a href="/es/roadmap">Roadmap</a></li>
+  </ul>
+</div>
+
+Las páginas de AMP son sólo páginas web con algunas características adicionales (y algunas limitaciones), haciéndolas consistentemente más rápidas y más amigables con el usuario final. Muchas de las mejores prácticas de esta guía son coherentes en las páginas que no son de AMP.
+
+Para obtener resultados óptimos, enfoque los anuncios en páginas de AMP de la misma manera que los anuncios en páginas que no son de AMP.
+
+<a class="button go-button" href="/es/docs/guides/ads/ads_getting_started">¡Empecemos!</a>
```
Comment (SidVal) [inline content/docs/guides/ads_on_amp@es.md]: Done. :+1:

File: content/docs/guides/ads_on_amp/ads_getting_started@es.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+[Aprende más sobre el placeholder](/es/docs/guides/responsive/placeholders#placeholders).
```
Comment (bpaduch) [inline content/docs/guides/ads_on_amp/ads_getting_started@es.md]: Can you put placeholder in code format? Like `placeholder`.

File: content/docs/guides/ads_on_amp/ads_getting_started@es.md (hunk 1 above)
Comment (bpaduch) [inline content/docs/guides/ads_on_amp/ads_getting_started@es.md]: Can you put fallback in code format? Like `fallback`.

File: content/docs/guides/ads_on_amp/ads_tips@es.md (hunk 3 above)
Comment (bpaduch) [inline content/docs/guides/ads_on_amp/ads_tips@es.md]: :+1:

File: content/docs/guides/ads_on_amp/ads_getting_started@es.md (hunk 1 above)
Comment (SidVal) [inline content/docs/guides/ads_on_amp/ads_getting_started@es.md]: Yes, of course.

Comment (SidVal) [inline content/docs/guides/ads_on_amp/ads_getting_started@es.md]: Yes, of course.

**Pull request merged**